app/scripts/metamask-controller.js

```diff
@@ -204,7 +204,6 @@ export default class MetamaskController extends EventEmitter {
     this.keyringController = new KeyringController({
       keyringTypes: additionalKeyrings,
       initState: initState.KeyringController,
-      getNetwork: this.networkController.getNetworkState.bind(this.networkController),
```

Comment:
This was expected by the KeyringController constructor and set on the instance, but never used after that. It has been fully removed in this PR: https://github.com/MetaMask/KeyringController/pull/76